### ISSUE TITLE
Standardization of input data versus scaling

### ISSUE THREAD

**Issue by jfelding** [jfelding/esn#8]
Currently, utils.preprocess() preprocesses (and postprocesses if argument `inverse=True` is passed). Of course, we use training set estimates to preprocess validation set, with no estimation on the latter. utils.preprocess() also does IMED-transformation (standardizing transform), but for scaling makes a call to utils.scale().

However, we may wan to standardize instead, center the data by deducting the mean, and using spread according to the standard deviation each pixel over time in the training set. That is done as such in 1D:
<img height="30" src="https://render.githubusercontent.com/render/math?math=x_{standard}=\cfrac{x-\mu_x}{\sigma_x}">
